### PR TITLE
Move nonlocal form factors in a separated directory

### PR DESCRIPTION
**Pull request by mreboud**
Solves #690

### CODE REVIEW

**Comment by mreboud** [comment]
I tried to move all the relevant files. The choice of files and the names can be discussed...

File: eos/Makefile.am [hunk 1]
```diff
@@ -11,6 +11,7 @@ SUBDIRS = \
 	models \
 	statistics \
 	form-factors \
+	nonlocal-form-factors \
```
Comment (dvandyk) [inline eos/Makefile.am:14]: Very good, before the rare-b-decays.

File: eos/nonlocal-form-factors/Makefile.am [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+libeosnonlocalformfactors_la_LDFLAGS = $(AM_LDFLAGS) $(GSL_LDFLAGS)
+libeosnonlocalformfactors_la_LIBADD = \
+	$(top_builddir)/eos/utils/libeosutils.la \
+	$(top_builddir)/eos/maths/libeosmaths.la \
+	$(top_builddir)/eos/models/libeosmodels.la \
+	$(top_builddir)/eos/form-factors/libeosformfactors.la \
+	-lboost_filesystem -lboost_system \
+	-lgsl -lgslcblas -lm
+
+include_eos_nonlocalformfactorsdir = $(includedir)/eos/rare-b-decays
```
Comment (dvandyk) [inline eos/nonlocal-form-factors/Makefile.am:38]: Typo, should be installed to ``$(includedir)/eos/nonlocal-form-factors``.

**Review by dvandyk — APPROVED** [review]
Only found one typo, see comments.
Approved once fixed.

File: eos/nonlocal-form-factors/Makefile.am (hunk 2 above)
Comment (mreboud) [reply eos/nonlocal-form-factors/Makefile.am:38]: Yes! Thanks :)

**Pull request merged**